src/graphic_model/engravers/lomse_chord_engraver.cpp

```diff
@@ -911,8 +911,6 @@ bool BeamedChordHelper::compute_stem_direction_for_chord(ImoNote* pBaseNote,
         fStemDown ? ++m_nDownForced : ++m_nUpForced;
         computedStem = (fStemDown ? k_computed_stem_forced_down
                                   : k_computed_stem_forced_up);
-        pChord->set_stem_direction(computedStem);
-        m_pStemsDir->push_back(computedStem);
```

Comment:
These lines are removed as the same two lines are already placed few lines later outside this if-else block. Therefore these lines were just adding a duplicated element to the `m_pStemDir` list.

Reply:
Yes! Good fix.